apps/dashboard/src/hooks/use-new-dashboard-opt-in.ts

```diff
@@ -0,0 +1,50 @@
+import { useUser } from '@clerk/clerk-react';
+import { NewDashboardOptInStatusEnum } from '@novu/shared';
+
+export function useNewDashboardOptIn() {
```

Comment:
1:1 hook as `useNewDashboardOptIn` in `web`. Not sure if putting this to `shared` would be good idea since it depends on `@clerk` package and we haven't stored hooks in shared yet